features/logout-testcase.feature

```diff
@@ -0,0 +1,13 @@
+Feature: logout
```

Comment:
In general looks great!  

Please adjust formatting as @monikakrawczak did. You can also work out together the way of formatting (using minuscules and majuscules, way of naming features and scenarios, etc.). This is just to be consistent in every .feature file. 

Please ping me on gitter, if you have any further questions!

Reply:
I've adjusted the formatting and capitalization to match Monika's testcases